### DISCUSSION TITLE
Vnom Export requires Simulation ID

### DISCUSSION THREAD

**Discussion by shpoudel** [GRIDAPPSD/gridappsd-forum#100] [API] [answered]
Hello Platform Team,

I was querying the nominal voltage using the Configuration file API (Vnom Export). I learned that the request needs to have a simulation ID as one of the parameters. However, I was expecting to get it with just a model ID similar to Ybus Export. 

@poorva1209

**Comment by poorva1209** [comment, marked as answer]
Yeah, the current implementation requires simulation id. I can change it to be based on the model id. There is already an issue on board for this https://github.com/gridappsd/goss-gridapps-d/issues/1431. I can make the change this week.
